app/agents/voice/automatic/tools/juspay/analytics.py

```
@@ -500,6 +500,78 @@ def get_average_ticket_payment_wise_by_time(
+
+        # Combine responses
+        combined_data = {
+            "refund_by_gateway": gateway_result.data,
+            "refund_by_payment_method": method_result.data,
+        }
+
+        logger.info(f"Combined data: {combined_data}")
+
+        await params.result_callback({"data": json.dumps(combined_data)})
+
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)